README.md

```diff
@@ -1 +1,72 @@
-# Bookstore
\ No newline at end of file
+# Bookstore
+
+# boiler-templet
+
+## Bookstore
+
+Created the Bookstore.
+
+# Bookstore
+> This is the first project of the Bookstore application. You will set up the environment and tools needed to develop a React application. In the following projects, you will develop the actual application.
+
+## Built With
+- HTML5
+- CSS
+- Javascript
+- React
+- Redux
+
+## How to use and run this project
+
+>To get a local copy up and running follow these simple example steps.
+
+- Clone this repository with
+
+git clone https://github.com/zairdon20/Bookstore.git using your terminal or command line.
+
+- Change to the project directory by entering :
+
+cd todo-list-project in the terminal.
+
+- run code . to open it in vscode
+- Install packages using the command npm install
+- run the project locally by entering the command npm start
+
+
+
+## Project Status
+This project is in progress.
+
+## Screenshoots
+![screenshot](./public/Screenshot-Bookstore4.png)
```

Comment:
- [ ] Your screenshot does not work, because there is no such file. I would suggest you add it or remove this section